composer.json

```diff
@@ -110,7 +110,7 @@
         "scssphp/scssphp": "v1.12.0",
         "setasign/fpdi": "^2.3.7",
         "setasign/tfpdf": "^1.33",
-        "shopware/conflicts": ">=0.2.0",
+        "shopware/conflicts": ">=0.3.0",
```

Comment:
the issue was now suddenly composer decided to prefer the latest twig 3.18 update, instead of conflicts 0.3 which conflicts twig >= 3.15

Now we force the update of the conflicts package, which leads to a downgrade of twig

@shyim I guess we had those types of issues before, any better ideas?